FriendsOfTYPO3/widgets
# [BUGFIX] Refactor of grunt with all packages

In the old lock file there was a security issue. Decided to refactor grunt complete to get rid of the issue, because braces packages was deep inside some packages without updates.

To test you should:
- Remove `build/node_modules` folder
- Run `yarn install`
- Run `yarn lint` --> no issues
- Remove the stylesheets in `public/Css` folder
- Run `yarn build`
- Check if both files where added back
- Extra feature: By running `grunt <name of widget in gruntfile.js>`. You can build and watch only one widget.

Closes: #13
